### ISSUE TITLE
Move today tab content

### ISSUE THREAD

**Issue by partly-igor** [weather-gov/weather.gov#971]
### Description
Updated 4/11: 
Following discussions with product, we are updating the design and organization of the content. [Figma prototype](https://www.figma.com/proto/0dSABt9zCqoknaKua6DHmP/NOAA---Weather.gov?page-id=4768%3A91967&type=design&node-id=4823-108688&viewport=104%2C449%2C0.07&t=SCR3XnwoascQ64zX-1&scaling=scale-down&starting-point-node-id=4823%3A108688&mode=design) for the update design. 

We are going to remove the the today tab, and instead place all forecast data and the weather story on one tab. The new set of tabs will be:
1. Alerts 
2. Current 
3. 7 Days (update name)

The weather story will be placed inside an accordion, and the accordion should be the first time item on the 7 Days tabs. The accordion should be collapsed by default. 

We will present all 7 days the same way, so we can remove the "extended forecast" heading, and change those items to be regular daily-item cards. 

We will also move the precipitation amounts table into the daily item cards, but that work will be captured in another ticket. 

This ticket should be done before starting on #1036 

~Based on our finding in R3 design is proposing updating the order of the content on the "Today" tab. We propose placing the forecast table first, followed by the precipitation amount, and then the weather story. This more closely matches with users expectations as we heard during our interviews.~

### Acceptance Criteria
- [ ] Remove the "Today" tab
- [ ] Rename "Daily" to "7 days"
- [ ] Place the weather story in an accordion
- [ ] Place the weather story accordion as the first item on the 7 days tab
- [ ] Include cards for the next 7 days (including today)
- [ ] Remove the "Extended forecast" heading
- [ ] Include the WFO Promo at the bottom of the page
- [ ] Update test as needed

**Comment by colinmurphy01** [comment]
Moving this to `blocked` until we discuss and decide on the best path forward.

**Comment by partly-igor** [comment]
Updated description and AC based on conversations in PO review (4/10) and eng-design cowork  (4/11)
